test/godeps/lxd-agent.list

```diff
@@ -33,6 +33,7 @@ github.com/canonical/lxd/lxd/events
 github.com/canonical/lxd/lxd/firewall
 github.com/canonical/lxd/lxd/firewall/drivers
 github.com/canonical/lxd/lxd/fsmonitor
+github.com/canonical/lxd/lxd/fsmonitor/drivers
```

Comment:
i'd like to understand why lxd-agent already "needs" fsmonitor, and whether we can get rid of both of these dependencies. Does the lxd-agent actually use fsmonitor for anything?